sdk/scala-sdk/src/main/resources/reference.conf

```diff
@@ -0,0 +1,67 @@
+akkaserverless {
```

Comment:
This will not work, since we include the reference.conf from javasdk dependency, with the same config properties.
Good news is that it's not needed at all.

Reply:
Cool will remove